<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>851</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>850</width>
      <height>850</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="put_flag_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>720</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background:  '#00008B'</string>
    </property>
    <property name="text">
     <string>Put a note &quot;?&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>831</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Wide Latin</family>
      <pointsize>50</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: 'white'</string>
    </property>
    <property name="text">
     <string>**SAPPER **</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>17</y>
      <width>1071</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Wide Latin</family>
      <pointsize>50</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: '#00008B'</string>
    </property>
    <property name="text">
     <string>**SAPPER **</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
